src/UI/Art/SpriteArtist/InfoView.java

```diff
@@ -53,81 +57,111 @@ public void render(Graphics g)
-    myG.fillRect(2*paneOuterBuffer, 2*paneOuterBuffer, paneHSize                    , paneVSize                    );
-    
-    // TODO: consider drawing all pages as one big image, so the user can scroll smoothly through it regardless of screen size
-    int drawingWidth = paneHSize - paneOuterBuffer;
-    switch (page.pageType)
-    {
-      case CO_HEADERS:
-        BufferedImage headersImage = CommanderOverlayArtist.drawAllCommanderOverlays(
-            myControl.getGame().commanders,
-            getDrawableMap(myControl.getGame()),
-            drawingWidth, paneVSize, thisCO);
-        myG.drawImage(headersImage, 2*paneOuterBuffer, 2*paneOuterBuffer, null);
-        break;
-      case GAME_STATUS:
-        if( null != thisCO && thisCO != statsCO )
-        {
-          statsCO = thisCO;
-          COStateInfo coStatus = new COStateInfo(getDrawableMap(myControl.getGame()), thisCO);
-          BufferedImage[] statusHalves = new BufferedImage[2];
-          // Draw assuming we don't have to wrap the text
-          statusHalves[0] = SpriteUIUtils.drawProse(coStatus.getFullStatusLabels());
-          statusHalves[1] = SpriteUIUtils.drawProse(coStatus.getFullStatusValues());
-          statsImage = SpriteUIUtils.joinBufferedImages(statusHalves, 3);
-        }
-        if( null != statsImage )
-          myG.drawImage(statsImage, 3 * paneOuterBuffer, 3 * paneOuterBuffer, null);
-        break;
-      case BASIC:
-        BufferedImage infoText = SpriteUIUtils.drawProseToWidth(page.info, drawingWidth);
-        myG.drawImage(infoText,3*paneOuterBuffer, 3*paneOuterBuffer, null);
-        break;
-    }
-    // Destroy stale state so we don't get into weird situations
-    if( page.pageType != PageType.GAME_STATUS )
-      statsCO = null;
+    myG.fillRect(2*PANE_OUTER_BUFFER, 2*PANE_OUTER_BUFFER, paneHSize                      , paneVSize                      );
+
+    int drawingWidth = paneHSize - PANE_OUTER_BUFFER;
+    if( pages != prevPages )
+      pageImage = renderPage(pages, drawingWidth);
+    prevPages = pages;
+
+    final int shiftDown = myControl.getShiftDown() * 2;
+    final int possibleShift = Math.max(0, pageImage.getHeight() - paneVSize);
+    int pixelShift = 0;
+    if( possibleShift > 0 )
+      pixelShift = (possibleShift + shiftDown % possibleShift) % possibleShift;
+
+    final int contentWidth  = Math.min(Math.abs(paneHSize - PANE_OUTER_BUFFER), pageImage.getWidth());
+    final int contentHeight = Math.min(Math.abs(paneVSize - PANE_OUTER_BUFFER), pageImage.getHeight());
+    // Draw the cropped page into our pane
+    myG.drawImage(pageImage.getSubimage(0, pixelShift, contentWidth, contentHeight), 3 * PANE_OUTER_BUFFER, 2 * PANE_OUTER_BUFFER, null);
 
     // Finally, draw our rendered image onto the window.
     g.drawImage(image, 0, 0, imageWidth*drawScale, imageHeight*drawScale, null);
   }
 
-  
+  private BufferedImage renderPage(ArrayList<InfoPage> pages, int drawingWidth)
```

Comment:
It would be nice to add arrows in the top- and bottom-right corners of the text pane to show that it can scroll up and down. Maybe disappear them if it all fits in one page.